### DISCUSSION TITLE
Does API login support HttpOnly Cookie?

### DISCUSSION THREAD

**Discussion by ganjarsetia** [sujit-baniya/fiber-boilerplate#28] [Q&A] [answered]
Hi, great work for the boilerplate!

I got [this](https://dev.to/guillerbr/authentication-cookies-http-http-only-jwt-reactjs-context-api-and-node-on-backend-industry-structure-3f8e) for secure authentication

![image](https://user-images.githubusercontent.com/3258090/196021203-0de8166b-0b2c-4801-bc4d-739bee2c6a0d.png)

Does your boilerplate support it?

**Comment by sujit-baniya** [comment, marked as answer]
Hi @ganjarsetia,

It does support HttpOnly. Since the boilerplate also uses Session and Cookies you can customized the boilerplate as per your requirement.
